packages/components/datepicker/src/Datepicker.tsx

```diff
@@ -1,27 +1,184 @@
-import React from 'react';
+import React, {
+  ChangeEventHandler,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
 import { cx } from 'emotion';
 import { CommonProps } from '@contentful/f36-core';
 
+import { format, isValid, parse, startOfDay, endOfDay } from 'date-fns';
 import { getStyles } from './Datepicker.styles';
 import { Calendar } from './Calendar';
+import { Popover } from '@contentful/f36-popover';
+import { DayPickerSingleProps } from 'react-day-picker';
+import FocusLock from 'react-focus-lock';
+import { TextInput, TextInputProps } from '@contentful/f36-forms';
+import { IconButton } from '@contentful/f36-button';
+import { CalendarIcon } from '@contentful/f36-icons';
+import { PopoverProps } from '@contentful/f36-popover/src';
 
-export type DatepickerProps = CommonProps;
+const DATE_FORMAT = 'dd/MM/yyyy';
```

Comment:
Does this format match how we're showing dates in products? We don't use `–` or `.`?

Reply:
Mostly we use such format: 
<img width="334" alt="image" src="https://user-images.githubusercontent.com/10744462/173573892-7eb06e48-260a-44e1-ae17-735be150d9e7.png">

But I agree that we have to double-check it with the design team. @damann what do you think?